web/src/components/input/Field.scss

```diff
@@ -16,6 +17,10 @@
     width: 100px;
     text-align: right;
     flex-shrink: 0;
+    display: flex;
```

Comment:
Do these 4 lines actually do anything? Cause the label is already right-aligned, and everything is vertically centered due to `align-items: center;` on `.field`, or am I missing something there?

Reply:
If these lines aren't here then any buttons you put with the label will also be right-aligned. This allows me to add more stuff to the label which gets left-alligned (like the parent-button)

Reply:
aka: it makes it work for things that aren't text.

Reply:
Nope, you're right. I seem to be mixing up two solutions. __removed__